packages/kit/src/packaging/test/fixtures/exports-replace/package.json

```diff
@@ -1,7 +1,7 @@
 {
 	"name": "exports-replace",
 	"version": "1.0.0",
-	"description": "default exports settings",
+	"description": "exports.behavior settings set to replace",
 	"exports": {
 		".": {
 			"import": "./index.js"
```

Comment:
I would change the export here to something else to test it really is overwritten. Right now the test result is that only `package.json` is appended.

Reply:
Ah yeah my bad, I got confused by what replaces what there 😄